src/core/terrainGeneration/LCG.js

```diff
@@ -0,0 +1,27 @@
+// Linear Congruential Generator; simple pseudo-random number generator
+// that will be used for 1D perline noise terrain generation
+// pseudo-random numbers are numbers that may seem random, but aren't
+export class LCG {
```

Comment:
AI generation at its finest. 
Reply to this one too to see if the comments on code change

Reply:
Nah all the terrain generation was basically from a tutorial/article, I wa just making bare comments for notes and understanding cus I did not make this algo from scratch (it is a pre existing one called perlin 1d noise generation). The hard thing was trying to adapt and actually understand the algorithm to pixijs.

Reply:
Niiice. You're gonna be cracked